x-pack/platform/plugins/shared/fleet/common/constants/mappings.ts

```diff
@@ -42,6 +42,7 @@ export const AGENT_POLICY_MAPPINGS = {
     unenroll_timeout: { type: 'integer' },
     updated_at: { type: 'date' },
     updated_by: { type: 'keyword' },
+    supports_agentless: { type: 'boolean' },
```

Comment:
not a mapping change. the objects here are used to validate properties that can be used in KQL filtering, so just adding `supports_agentless` to this allow list